### PR TITLE
DOCS-11380-Added redeploying scenarios to the Delete Applications section

### PR DESCRIPTION
**Pull request by luanamulesoft**

### CODE REVIEW

**Review by martinmulesoft — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/managing-applications-on-your-own-servers.adoc [hunk 1]
```diff
@@ -78,7 +78,17 @@ To delete a deployed application from a server, simply click on the arrow next t
 image::managing-applications-on-your-own-servers-96463.png[]
 
 [NOTE]
-Note that the server must be running for this option to be available. In case of a Cluster or Server Group, at least one of its nodes must be running.
+Note that the server must be running for this option to be available. In the case of a Cluster or Server Group, at least one of its nodes must be running.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/managing-applications-on-your-own-servers.adoc]:
```suggestion
The server must be running for this option to be available. In the case of a Cluster or Server Group, at least one of its nodes must be running.
```

File: modules/ROOT/pages/managing-applications-on-your-own-servers.adoc [hunk 2]
```diff
@@ -78,7 +78,17 @@ To delete a deployed application from a server, simply click on the arrow next t
...
+
+=== Redeploying Deleted Hybrid Applications
+
+For applications deployed on premises, a couple of scenarios may arise which will trigger specific behaviours by Runtime Manager:
```
Comment (sarathecoubian) [inline modules/ROOT/pages/managing-applications-on-your-own-servers.adoc]:
```suggestion
For applications deployed on-premises, a couple of scenarios may arise which trigger specific behaviors by Runtime Manager:
```

File: modules/ROOT/pages/managing-applications-on-your-own-servers.adoc [hunk 3]
```diff
@@ -78,7 +78,17 @@ To delete a deployed application from a server, simply click on the arrow next t
...
+
+* If you manually delete one or several application folders from the server while Mule is not running, once the server re-syncs, Runtime Manager will notice the missing applications, force you to download the missing application or applications again into your server, and automatically re-deploy it.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/managing-applications-on-your-own-servers.adoc]:
```suggestion
* If you manually delete one or several application folders from the server while Mule is not running, once the server re-syncs, Runtime Manager notices the missing applications, forces you to download the missing application or applications again into your server, and automatically re-deploys it.
```

File: modules/ROOT/pages/managing-applications-on-your-own-servers.adoc [hunk 4]
```diff
@@ -78,7 +78,17 @@ To delete a deployed application from a server, simply click on the arrow next t
...
+[NOTE]
+Only deleting from the UI or via REST will permanently remove the application.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/managing-applications-on-your-own-servers.adoc]:
```suggestion
Only deleting from the UI or via REST permanently removes the application.
```

File: modules/ROOT/pages/managing-applications-on-your-own-servers.adoc [hunk 5]
```diff
@@ -78,7 +78,17 @@ To delete a deployed application from a server, simply click on the arrow next t
...
+
+* When a new server is added into a Cluster or Server Group, even when no other server in that Group or Cluster is running, Runtime Manager will automatically deploy any applications into that Server Group or Cluster.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/managing-applications-on-your-own-servers.adoc]:
```suggestion
* When a new server is added into a Cluster or Server Group, even when no other server in that Group or Cluster is running, Runtime Manager automatically deploys any applications into that Server Group or Cluster.
```

**Review by sarathecoubian — DISMISSED** [review]
A few suggestions!

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**